packages/braintree-integration/src/braintree-paypal-ach/braintree-paypal-ach-payment-strategy.ts

```diff
@@ -24,7 +24,7 @@ import { WithBraintreePaypalAchPaymentInitializeOptions } from './braintree-payp
 
 export default class BraintreePaypalAchPaymentStrategy implements PaymentStrategy {
     private usBankAccount?: BraintreeBankAccount;
-    private mandateText? = '';
+    private getMandateText?: () => string | undefined;
```

Comment:
I think we can skip adding `undefined`, because getMandateText is an optional param/variable.